drivers/dummy/rasterizer_dummy.h

```diff
@@ -725,20 +725,25 @@ class RasterizerStorageDummy : public RasterizerStorage {
 	}
 
 	bool free(RID p_rid) {
-
 		if (texture_owner.owns(p_rid)) {
 			// delete the texture
 			DummyTexture *texture = texture_owner.get(p_rid);
 			texture_owner.free(p_rid);
 			memdelete(texture);
-		}
-
-		if (mesh_owner.owns(p_rid)) {
+		} else if (mesh_owner.owns(p_rid)) {
 			// delete the mesh
 			DummyMesh *mesh = mesh_owner.getornull(p_rid);
 			mesh_owner.free(p_rid);
 			memdelete(mesh);
+		} else if (lightmap_capture_data_owner.owns(p_rid)) {
+			// delete the lightmap
+			LightmapCapture *lightmap_capture = lightmap_capture_data_owner.getornull(p_rid);
+			lightmap_capture_data_owner.free(p_rid);
+			memdelete(lightmap_capture);
+		} else {
+			return false;
```

Comment:
One thing I wasn't sure of, is if this dummy storage was always returning `true` on purpose.

I figured it wasn't, so I fixed that. But if someone has deeper knowledge about why it was originally always returning true, please let me know.

Reply:
The main question is whether `free()` will be called on objects which are not owned by RasterizerDummy, and should simply be skipped as a no-op with a positive result (remember it's a dummy - a hack - so it's not expected to handle everything, only the minimum necessary for headless use).
If it doesn't spew tons of errors when you export after this change, I guess it might be OK.

Reply:
I need to do a little more checking, but it almost appeared to have fixed some other leaks.

Originally it was complaining about leaking some Canvas Items, and now it doesn't appear to be. Is it possible that the RID is passed to other storage objects if false is returned? And thus by passing true for everything, it was preventing other RIDs from getting cleaned up?

Reply:
Well at least the GLES2 and GLES3 implementations seem to do the same, so this should be fine.